source/assets/stylesheets/base/_variables.scss

```diff
@@ -34,3 +43,7 @@ $base-border: 1px solid $base-border-color;
 // Forms
 $form-box-shadow: inset 0 1px 3px rgba(#000, 0.06);
 $form-box-shadow-focus: $form-box-shadow, 0 0 5px adjust-color($action-color, $lightness: -5%, $alpha: -0.3);
+
+// Shadows
+$base-box-shadow: 0 0 32px 0 rgba(black, 0.25);
```

Comment:
You could do `$box-shadow-size` or something like that to store `0 0 32px 0` since it repeats.